### ISSUE TITLE
CONTRIBUTING.md missing

### ISSUE THREAD

**Issue by giovanism** [laymonage/TarungLab#4]
Add guidelines for issues, commit messages, and PRs.

So https://github.com/laymonage/TarungLab/pull/3 won't happen again

## Reference
https://github.com/blog/1184-contributing-guidelines
https://github.com/coala/coala/blob/master/CONTRIBUTING.rst

## Proposal
1. Source code (python)
   - [ ] Passed pylint source code analyzer (except C0103 for the time being)
   - [ ] Maximum line length is set to 79
   - [ ] File naming format `(Lab)0[1-99]_[A-F](\+[A-F])?(_s[1-9])?(\.py)`
   - [ ] ...

2. Problem sets
   - [ ] Include only the necessary instruction and hint to complete the problem
   - [ ] Verbatim with necessary formatting adjustment
   - [ ] A footer
   - [ ] File naming format `(Lab)0[1-99]_[A-F](\+[A-F])?(\.md)`
   - [ ] ...

3. Commit messages
   - [ ] First line < 50 chars
   - [ ] ...

4. PRs
5. Issues

**Comment by laymonage** [comment]
Yep, we need `CONTRIBUTING.md` in this repo. I like your proposal so far.
Some points I can think of right now:
- For PyLint, we have a problem in class F's turtle solutions since they don't have any turtle object instantiation, which leads to an E1101 error. We're gonna use CI, and I think we can lint those solutions individually in CI (I'm not a fan of adding lint exceptions inside source codes).
- Specify footer format for problem sets.
- Add details about how images should be included.
- Commit messages should use `Sentence case`. See previous commits for examples.
- PRs' rules are pretty much the same as commit messages', but they're more flexible (I think?).

**Comment by giovanism** [comment]
@laymonage , I think I can change class F solution to conform with PyLint's rule. Another thing to mention, would it be better for python source code filename format in lower case, to conform PyLint module name rule?

**Comment by laymonage** [comment]
I didn't know PyLint has rules for module names. Well, if you think so, feel free to send a PR that fixes it.
